empress/support_files/templates/empress-template.html

```diff
@@ -209,6 +215,16 @@ <h3 id="menu-sm-header">Sample Presence Information</h3>
             $(".needs-feature-metadata").addClass("hidden");
         }
 
+        var statsButton = document.getElementById("stats-btn");
+        statsButton.shearUpdate = () => {
+          statsButton.classList.remove("unpopulated");
+          if (!statsButton.classList.contains("unpopulated")) {
+            sPanel.populateTreeStats();
+          }
```

Comment:
```suggestion
          sPanel.populateTreeStats();
          statsButton.classList.remove("unpopulated");
```

I don't think there is ever a case where the conditional in the `if` statement isn't true, right? Since we remove the `unpopulated` class immediately beforehand.

Also, it may be worth taking these two lines of code and making them into a function since they're reused in this file... but not a big deal.